src/main/java/com/demo/controller/DemoApplicationController.java

```diff
@@ -14,6 +14,8 @@
 @RequiredArgsConstructor
 public class DemoApplicationController {
 
+    //Updated
+
     private final PlansService plansService;
 
     @GetMapping(value = "/Demo/getPlans/{planType}")
```

Comment:
Can we add the swagger description of what this endpoint does?

Reply:
Added swagger documentation for all the required endpoints and Models.